Cargo.toml

```diff
@@ -193,65 +193,51 @@ clippy.wildcard_dependencies = "deny"
 [workspace]
 resolver = "2"
 members = [
-    "cli",
-    "client",
-    "client_cli",
-    "config",
-    "config/base",
-    "core",
-    "core/test_network",
-    "crypto",
-    "data_model",
-    "genesis",
-
-    "logger",
-    "p2p",
-
-    "futures",
-    "futures/derive",
-
-    "primitives",
-    "primitives/derive",
-    "primitives/numeric",
-
-    "ffi",
-    "ffi/derive",
-
-    "schema",
-    "schema/gen",
-    "schema/derive",
-
-    "smart_contract",
-    "smart_contract/derive",
-    "smart_contract/utils",
-
-    "smart_contract/trigger",
-    "smart_contract/trigger/derive",
-
-    "smart_contract/executor",
-    "smart_contract/executor/derive",
-    "smart_contract/executor/data_model",
-    "smart_contract/executor/data_model/derive",
-
-    "telemetry",
-    "test_samples",
-
-    "tools/swarm",
-    "tools/kagami",
-    "tools/kura_inspector",
-    "tools/parity_scale_cli",
-    "tools/wasm_builder_cli",
-    "tools/wasm_test_runner",
-
-    "torii",
-    "torii/const",
-
-    "version",
-    "version/derive",
-
-    "wasm_codec",
-    "wasm_codec/derive",
-    "wasm_builder",
+    "crates/irohad",
+    "crates/iroha",
+    "crates/iroha_cli",
+    "crates/iroha_config",
+    "crates/iroha_config_base",
+    "crates/iroha_core",
+    "crates/iroha_test_network",
+    "crates/iroha_crypto",
+    "crates/iroha_data_model",
+    "crates/iroha_genesis",
+    "crates/iroha_logger",
+    "crates/iroha_p2p",
+    "crates/iroha_futures",
+    "crates/iroha_futures_derive",
+    "crates/iroha_primitives",
+    "crates/iroha_primitives_derive",
+    "crates/iroha_numeric",
+    "crates/iroha_ffi",
+    "crates/iroha_ffi_derive",
```

Comment:
why do we want to make derive crates adjacent?

Reply:
There is no benefit to nesting crates, it only adds mental overhead. Cargo doesn't have nested workspaces